src/views/ErrorAlert.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/require-default-props */
```

Comment:
#### Critical
- Don't do this

Reply:
Alright, I have removed that. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198142%2Fsignup/src%2Fviews%2FErrorAlert.jsx